packages/facet-filter/src/components/inputs/checkbox/CheckboxView.js

```diff
@@ -33,15 +35,16 @@ const CheckBoxView = ({
 }) => {
   const {
     name,
+    customName,
     subjects,
+    customSubjects,
     isChecked = false,
     index,
     section,
     tooltip,
-    label,
   } = checkboxItem;
   const indexType = index % 2 === 0 ? 'Even' : 'Odd';
-  const checkedSection = `${section}`.toLowerCase().replace(' ', '_');
+  const checkedSection = `${section}`.toLowerCase().replace(/\ /g, '_');
```

Comment:
Same for this one, Is this going to be specific to ICDC? If that's the case this regular expression can be prop and if none passed we can use the default one based on bento.